builder/shellParser_test.go

```diff
@@ -0,0 +1,51 @@
+package builder
```

Comment:
Actually I have one minor objection. I don't think the project uses camelCase anywhere for filenames and I wouldn't want to rock the boat. :P

-Erik

> On Mar 20, 2015, at 7:11 PM, Erik Hollensbe <github@hollensbe.org> wrote:
> 
> Nah, it's fine, I was just confused. Thanks for responding. No objections from me.
> 
> -Erik
> 
>> On Mar 20, 2015, at 7:06 PM, Doug Davis <notifications@github.com <mailto:notifications@github.com>> wrote:
>> 
>> In builder/shellParser_test.go <https://github.com/docker/docker/pull/10431#discussion_r26887689>:
>> 
>> > @@ -0,0 +1,51 @@
>> > +package builder
>> not 100% sure what I was thinking at that exact moment :-) but I probably put it in the builder because shellParser (which is probably not the best name for it) isn't used during the parsing phase - its actually used during the env-var substitution phase. Perhaps I should have called it "shellWordProcessor" instead - dunno.
>> 
>> —
>> Reply to this email directly or view it on GitHub <https://github.com/docker/docker/pull/10431/files#r26887689>.
>> 
>